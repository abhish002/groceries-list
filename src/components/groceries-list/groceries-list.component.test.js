import React from 'react';
import { create } from 'react-test-renderer';
import GroceriesList from './groceries-list.component';

describe('groceries list component', () => {
  it('matches the snapshot', () => {
    const groceriesList = create(<GroceriesList />);
    expect(groceriesList.toJSON()).toMatchSnapshot();
  });
});