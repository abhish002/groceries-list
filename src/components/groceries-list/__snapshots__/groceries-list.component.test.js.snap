// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`groceries list component matches the snapshot 1`] = `
<div
  className="groceries-list"
/>
`;
